scm-ui/ui-webapp/src/repos/components/form/RepositoryForm.tsx

```diff
@@ -22,28 +22,18 @@
  * SOFTWARE.
  */
 import React, { FC, useCallback, useEffect, useState } from "react";
-import styled from "styled-components";
 import { useTranslation } from "react-i18next";
+import styled from "styled-components";
 import { ExtensionPoint } from "@scm-manager/ui-extensions";
 import { IndexResources, Repository, RepositoryType, CUSTOM_NAMESPACE_STRATEGY } from "@scm-manager/ui-types";
 import { Checkbox, Level, Select, SubmitButton } from "@scm-manager/ui-components";
 import NamespaceAndNameFields from "../NamespaceAndNameFields";
 import RepositoryInformationForm from "../RepositoryInformationForm";
 
-const CheckboxWrapper = styled.div`
-  margin-top: 2em;
+const FlexibleDiv = styled.div`
```

Comment:
You can set flex-grow and flex-shrink with bulma classes.

Reply:
During the conversion I noticed that the whole construct is not responsive, so I replaced it with .column.